Assets/_Project/Code/Models/ListaOdznak.cs

```diff
@@ -0,0 +1,29 @@
+﻿public class ListaOdznak
+{
+    private List<Odznaka> odznaki;

```

Comment:
To musi być oznaczone przez Json property inaczej nie zapiszą nam sie odznaki wraz z drużyną.